core/dbt/contracts/graph/manifest.py

```diff
@@ -763,10 +753,10 @@ def writable_manifest(self):
             parent_map=backward_edges,
         )
 
-    def to_dict(self, omit_none=True, validate=False):
-        return self.writable_manifest().to_dict(
-            omit_none=omit_none, validate=validate
-        )
+    # TODO: this was to_dict(). Ensure that the right method is called
```

Comment:
Is this a TODO for this PR, or for the future?

Reply:
Oops. That was something I need to check out for this PR. Will investigate.

Reply:
I've modified that to use a __pre_serialize__ callout.